Comb_Sort/README.md

```diff
@@ -0,0 +1,132 @@
+
+
+![Comb_sort_demo](https://user-images.githubusercontent.com/28682701/56884536-c1704380-6a87-11e9-9cf9-fb8cba477a53.gif)
+
+Comb sort is an improvement of the bubble sort algoritham.In the comb sort,the items are sorted in a specific gap. 
+After completing each phase,the gap decreased.
+
+With compared to the bubble sort this sorting algoritham remove the worst-case scenario and improved on the time complexity of Bubble sort.
+
+
+
+**Example**
+
+Let's arrange the numbers in ascending order using Comb sort.
+
+Unsorted List
+
+![Screenshot (35)](https://user-images.githubusercontent.com/28682701/56882906-b666e480-6a82-11e9-94f4-b3446f13ebf6.png)
+
+-Iteration 1
+
+Gap=8 (number of elements in the array/ 1.3)
+
+![Screenshot (36)](https://user-images.githubusercontent.com/28682701/56883448-56713d80-6a84-11e9-97ad-32337c030b64.png)
+
+
+The distance of 8 from the first element 5 to the next element, leads to the element of 2.These numbers are not in the right order,so they have to swap.Then the second element compared with the last element,they are not in the order so again swap is required.
+
+
+-Iteration 2
+
+New Gap Value = 8/1.3 = 6
+
+![Screenshot (38)](https://user-images.githubusercontent.com/28682701/56883662-fcbd4300-6a84-11e9-9a56-74f90a468644.png)
+
+
+-Iteration 3
+
+New Gap Value = 6/1.3 = 4
+
+![Screenshot (41)](https://user-images.githubusercontent.com/28682701/56883774-4efe6400-6a85-11e9-9287-b78a046c466b.png)
+
+
+
+-Iteration 4
+
+
+New Gap Value = 4/1.3 = 3
+
+![Screenshot (42)](https://user-images.githubusercontent.com/28682701/56883861-8f5de200-6a85-11e9-8ffe-f2b730804a00.png)
+
+
+-Iteration 5
+
+New Gap value = 3/1.3 = 2
+
+![Screenshot (43)](https://user-images.githubusercontent.com/28682701/56883958-de0b7c00-6a85-11e9-8d56-2497dbe4b89c.png)
+
+-Iteration 6
+
+New Gap Value = 2/1.3 = 1
+
+![Screenshot (44)](https://user-images.githubusercontent.com/28682701/56884136-76a1fc00-6a86-11e9-9c5a-176c1af7f025.png)
+
+***Algoritham***
+
+1. Calculate the Gap value.
+2. Iterating and swapping.
```

Comment:
Slightly elaborate on the 2nd point on the conditions for swapping

Reply:
@singh-shreya6  I have updated the file.could you kindly review it ?